pkg/apis/iot/v1alpha1/types_config.go

```diff
@@ -148,6 +149,7 @@ type ExternalCacheNames struct {
 type CommonServiceConfig struct {
 	Container *ContainerConfig      `json:"container,omitempty"`
 	Java      *JavaContainerOptions `json:"java,omitempty"`
+	corev1.SecurityContext
```

Comment:
Is this needed when it is set on the device registry config itself?

Reply:
It was the remains of an earlier idea. Removed.